CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
-cmake_minimum_required(VERSION 2.6)
+cmake_minimum_required(VERSION 3.0)
+project(cmake VERSION 1.0)
```

Comment:
Pls add the description field as well, it is also used in the pc.in file

Reply:
No .pc.in file in this package. But I'm adding it anyways.